plugin/pom.xml

```diff
@@ -43,7 +43,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>20.0</version>
+      <version>25.0-android</version>
```

Comment:
How about using [26.0-jre](https://search.maven.org/artifact/com.google.guava/guava/26.0-jre/bundle) and only supporting java 8+?